### PR TITLE
Testing: dissuade the use of page.has_content

### PR DESCRIPTION
**Pull request by rstacruz**
Controversial on this one, Jason was against it, but what do you think Vic?

### CODE REVIEW

**Comment by victorsolis** [comment]
I think `have_content` if fine **only** if you're sure na there will be no other occurrence nung string on the page. Otherwise, you're right. Mas okay yung may context.

**Comment by victorsolis** [comment]
Also, can't merge.

**Comment by rstacruz** [comment]
conflicts resolved

File: practices/testing.md [hunk 1]
```diff
@@ -374,32 +387,27 @@ Don't.
...
-- Never hard code emails and similar fields that must be unique. Use `sequence`.
+- Never hard-code emails and similar fields that must be unique. Use `sequence`.
 
   ```ruby
   factory :user do
-    sequence :email do |index|
-      "user-#{index}@email.com"
-    end
-
+    sequence(:email) { |i| "user-#{i}@email.com" }
```
Comment (victorsolis) [inline practices/testing.md]: We should probably mention na you can put all your sequences in a separate file to avoid duplication.

**Pull request merged**